src/main/java/de/retest/recheck/ui/diff/ElementDifference.java

```diff
@@ -131,7 +131,7 @@ public static String getSumIdentifier( final Collection<ElementDifference> diffe
 	}
 
 	public boolean hasAttributesDifferences() {
-		return attributesDifference != null;
+		return attributesDifference != null && attributesDifference.size() > 0;
```

Comment:
These `null` and `empty()` checks are getting out of control … 🤯

Reply:
Yes, sadly that's how our code works ... . I'd prefer the latter only and have something with `AttributesDifference.empty()` but that would be a bit of a rewrite.

Reply:
Sure, I know it's not your fault and we cannot easily change this.